src/mobile-ui-react/ModalEntryFormDialog.tsx

```diff
@@ -260,10 +265,12 @@ export function ModalEntryFormDialog(props: ModalEntryFormDialogProps) {
     // https://reactjs.org/docs/forms.html#controlled-components
     const input = e.currentTarget;
 
-    const valueContainsForbiddenChars = fields[index].forbiddenCharacters?.some((x) => input.value?.includes(x));
+    const rawForbiddenCharacters = fields[index].forbiddenCharacters;
+    const forbiddenCharacters = typeof rawForbiddenCharacters === "string" ? [...rawForbiddenCharacters] : rawForbiddenCharacters;
```

Comment:
When given an array you can join them all together to get your final array:

```suggestion
    const forbiddenCharacters = Array.isArray(rawForbiddenCharacters) ? [...rawForbiddenCharacters.join()] : [...rawForbiddenCharacters];
```

Reply:
I suggested to Valdas that I just remove `string[]` as a possible type for `forbiddenCharacters`. Your change appears to just make it so that if they include a multi-character entry in the array, it will get split into multiple separate one-character entries, and I don't think that is actually an improvement, since the presence of a multi-character entry is probably an error in the first place. Having said that, things don't work right with certain complicated Unicode sequences, since `[...<someString>]` ends up messing things up. So perhaps I should just leave it as it is, and update the `forbiddenCharacters` comment to indicate that if something like 🤷🏽‍♀️ is needed in their `forbiddenCharacters` list, they must use the array of strings.

Reply:
I updated the `forbiddenCharacters` comment.